ocean_provider/validation/algo.py

```diff
@@ -359,18 +419,18 @@ def validate(self):
             return False
 
         if self.service.type not in ["access", "compute"]:
-            self.error = "Services in input can only be access or compute."
+            self.error = "service_not_access_compute"
             return False
 
         if self.service.type != "compute" and self.index == 0:
-            self.error = "Service for main asset must be compute."
+            self.error = "main_service_compute"
             return False
 
         asset_urls = get_service_files_list(
             self.service, self.provider_wallet, self.asset
         )
         if self.service.type == "compute" and not asset_urls:
-            self.error = "Services in input with compute type must be in the same provider you are calling."
+            self.error = "compute_services_not_in_same_provider"
```

Comment:
Based on the previous comment, not really sure what would be the resource and the action here. Maybe `compute_services.not_in_same_provider`